testassets/InteropTestsGrpcWebWebsite/Tests/custom-environment.js

```diff
@@ -1,6 +1,9 @@
-// custom-environment.js
+// For some reason the expect-puppeteer require is returning a module rather than the function.
+// Helper function unwraps the module and returns the inner function.
+function _interopDefault(ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
+
 const PuppeteerEnvironment = require('jest-environment-puppeteer');
-const expect = require('expect-puppeteer');
+const expect = _interopDefault(require('expect-puppeteer'));
```

Comment:
```suggestion
import expect from 'expect-puppeteer'
```

Can we use module imports here instead?

Reply:
I tried that but got an error along the lines of "can't import outside of a module"

Reply:
Hmm, I didn't do exactly what you suggested: [`554f1bc` (#1615)](https://github.com/grpc/grpc-dotnet/pull/1615/commits/554f1bc48d494e95c8c42f10a2cdc28fca03453c)

I'll give it a try

Reply:
To get import to work the package type has to be module which means lots of other changes. The current fix probably isn't better but it is simpler.